### PR TITLE
K8SPSMDB-232: Add smart-update test

### PR DESCRIPTION
**Pull request by blez**

### CODE REVIEW

**Review by delgod — CHANGES_REQUESTED** [review]
Could you please add all the new tests from the main branch in this PR?

**Comment by blez** [comment]
@delgod, done

File: cloud/jenkins/psmdb_operator_minikube.groovy [hunk 1]
```diff
@@ -226,11 +226,22 @@ pipeline {
                     runTest('init-deploy')
                     runTest('limits')
                     runTest('scaling')
+                    runTest('monitoring')
```
Comment (hors) [inline cloud/jenkins/psmdb_operator_minikube.groovy:229]: Please do not add these tests for minikube. These tests do not work on minikube @cap1984  handle it.

File: cloud/jenkins/psmdb_operator_aws_openshift-4.groovy [hunk 2]
```diff
@@ -220,18 +220,28 @@ pipeline {
...
                 runTest('one-pod')
                 runTest('monitoring')
+                runTest('monitoring-2-0')
                 runTest('arbiter')
                 runTest('service-per-pod')
                 runTest('liveness')
                 runTest('users')
            }
         }
+        stage('E2E SelfHealing') {
```
Comment (hors) [inline cloud/jenkins/psmdb_operator_aws_openshift-4.groovy:238]: the same for these tests , It does not work for for openshift

**Review by hors — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Pull request merged**